src/main/java/com/epam/javacourses2016/task19/SolverTask19.java

```diff
@@ -17,8 +17,36 @@
      * @param numberLaps Количество кругов.
      * @return Количество осуществленных обгонов.
      */
-    int getNumberOvertaking(Set<Car> cars, long lengthLap, int numberLaps) {
-        //TODO
-        return 0;
+    int getNumberOvertaking(Set<Car> cars, long lengthLap, long numberLaps) {
+        int count = 0;
+        for (Car car1 : cars) {
+            for (Car car2 : cars) {
+                count += getOvertake(car1, car2, lengthLap, numberLaps);
```

Comment:
too slow) :snail: 
You simulate a race for each pair of cars instead of the single race simulation for everyone